openslides_backend/models/motion_comment.py

```diff
@@ -0,0 +1,30 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionComment(Model):
+    """
+    Model for motion comments-
+
+    There are the following reverse relation fields:
+        TODO
+    """
+
+    # TODO: Add reverse relation fields to docstring.
+
+    collection = Collection("motion_comment")
+    verbose_name = "motion_comment"
+
+    id = fields.IdField(description="The id of this motion comment.")
+    comment = fields.TextField(description="The comment of this motion comment.")
+    motion_id = fields.ForeignKeyField(
+        description="The id of the motion of this motion comment.",
+        to=Collection("motion"),
+        related_name="comment_ids",
+    )
+    section_id = fields.ForeignKeyField(
+        description="The id of the section of this motion comment.",
+        to=Collection("motion_comment_section"),
```

Comment:
This works even though the model doesn't exist yet? Interesting. @normanjaeckel I think we should catch this and throw an error in case of typos

Reply:
I do not know and I prefer using auto generated models.py in the future. See our coming emails ...